Lib/multiprocessing/forkserver.py

```diff
@@ -227,7 +230,10 @@ def sigchld_handler(*_unused):
                     with listener.accept()[0] as s:
                         # Receive fds from client
                         fds = reduction.recvfds(s, MAXFDS_TO_SEND + 1)
-                        assert len(fds) <= MAXFDS_TO_SEND
+                        if len(fds) > MAXFDS_TO_SEND:
+                            raise RuntimeError(
```

Comment:
I don't think this one is necessary.

Reply:
Will get back to you on this one - need to put together lecture for tomorrow...

Reply:
Why would it not be necessary?

Reply:
Because otherwise would probably a bug in the underlying `recvmsg` implementation, which I think doesn't need to be guarded by a nice error message.

Reply:
I'm actually more worried about detecting what bug is happening - see [here](https://travis-ci.org/drallensmith/neat-python/jobs/262839337) for why, noting that it's with a nightly build thus has the first patch in it.

Reply:
I see, fair enough.